openlibrary/plugins/openlibrary/pages/config_edition.page

```diff
@@ -50,85 +97,380 @@
+        "Printer",
+        "Producer",
+        "Production Assistant",
         "Production Controller",
         "Production Coordinator",
-        "Research",
-        "Senior Editor",
-        "Director",
-        "Musical Director",
+        "Production Editor",
+        "Project Coordinator",
+        "Project Editor",
+        "Project Team Leader",
+        "Reading Director",
+        "Recenzent",
+        "Recipe Tester",
         "Recording Producer",
-        "Producer",
-        "Collected by",
+        "Recording Studio",
+        "Redactor",
+        "Redakteur",
+        "Researcher",
+        "Research Director",
         "Reviewer",
-        "Co-Authors",
-        "collection",
-        "Information Officer",
-        "Research Dir.",
+        "Revis\u00e3o",
         "Revised by",
-        "Notes by",
+        "Script",
+        "Secretaria editorial",
         "Selected by",
-        "Copyright",
-        "Photo Scanning Specialist",
-        "printer",
-        "grunt",
-        "hallucinator",
-        "smallchange artist",
-        "Advisory Editor",
-        "Researcher"
+        "Senior Editor",
+        "Soloist",
+        "Sponsor",
+        "Stylist",
+        "Technical draftsman",
+        "Technical Editor",
+        "Tests and evaluations",
+        "Thanks",
+        "Titelillustration",
+        "Translated for",
+        "Typography",
+        "Umschlagentwurf",
+        "Vorauskorrektor",
+        "Vorwort",
+        "Web Programming & Design",
+        "Woodcuts",
+        "Writer",
+        "Brand Manager",
+        "Editorial Manager",
+        "Inspirador",
+        "Songs translated by",
+        "Book Design",
+        "Essayist",
+        "Publishing Director",
+        "Science Editor",
+        "Lithography",
+        "Series General Editor",
+        "Editorial Board Member",
+        "Publisher",
+        "Board of Consultants",
+        "Marketing Manager",
+        "Managing Editor",
+        "Scientific advisor",
+        "Retold by",
+        "interviewee",
+        "Cover photograph",
+        "Image editor",
+        "Lektor",
+        "Acquisitions Coordinator",
+        "Computer Designer",
+        "Series Design",
+        "Map",
+        "Typografische Gestaltung",
+        "Premessa",
+        "Epigraph",
+        "Reviewers"
     ],
     "identifiers": [
+        {
+            "url": "https://alkindi.ideo-cairo.org/controller.php?action=SearchNotice&noticeId=@@@",
+            "website": "https://www.ideo-cairo.org/",
+            "notes": "",
+            "name": "dominican_institute_for_oriental_studies_library",
+            "label": "Al Kindi"
+        },
+        {
+            "url": "https://www.alibris.com/booksearch?qwork=@@@",
+            "notes": "",
+            "name": "alibris_id",
+            "label": "Alibris ID"
+        },
         {
             "url": "https://www.amazon.com/gp/product/@@@",
             "name": "amazon",
-            "label": "Amazon ASIN"
```

Comment:
I find calling it an ASIN is helpful, makes it clear that the value is an id, not just part of a URL

Reply:
I have a slight preference for Amazon.com over Amazon ASIN, but I just copied what I was given.

Reply:
One ASIN is used for the identical edition at all amazon sites: .com, .ca, .co.uk, .de, .fr, etc. That said, users will usually only have amazon accounts on one or two domains, principally their home country's domain.